isc24/CernVM-FS/abstract.tex

```diff
@@ -1,3 +1,27 @@
-CernVM-FS, the CernVM File System (also known as CVMFS), is a file distribution service that is particularly well suited to distribute software installations across a large number of systems world-wide in an efficient way.
+CernVM-FS is a file distribution service that is particularly well suited to distribute software installations across a large number of client systems in an efficient way.
 
-From an end user perspective, files in a CernVM-FS repository are available read-only via a subdirectory in /cvmfs, with a user experience similar to that of an on-demand streaming service for music or video, but then (mainly) applied to software installations.
+It provides several interesting features, including automatic and transparent on-demand downloading and updating of repository contents, multi-level caching, de-duplication of files, compression of data, and automatic verification of data integrity.
+From an end user perspective, files in a CernVM-FS repository are available read-only, with a user experience similar to that of an on-demand streaming service for music or video, but then (mainly) applied to software installations.
+
+CernVM-FS is a compelling solution for Continuous Delivery of software in HPC environments, since it has been proven to scale to billions of files and tens of thousands of clients. Once a software installation has been ingested in a CernVM-FS repository on a centrally managed server, it is
+automatically synchronized to a network of mirror servers and associated proxy servers.
+After a CernVM-FS repository has been made accessible on a system, no subsequent action must be taken by system administrators or end users to obtain updates to the contents of the repository, since that is done fully automatically by the client component of CernVM-FS via the network of mirror and proxy servers.
+
+This tutorial will introduce CernVM-FS through hands-on exercises and demos, and will cover specific aspects that are relevant to using it in an HPC context, including HPC-specific configuration, troubleshooting, performance, and containers. The European Environment for Scientific Software
+Installations (EESSI) will serve as example CernVM-FS repository.
```

Comment:
```suggestion
Installations (EESSI) will serve as an example CernVM-FS repository.
```

Reply:
Eek 251!

Reply:
I think it works without the `an` just fine?